pkg/specgen/generate/container.go

```diff
@@ -483,5 +488,17 @@ func ConfigToSpec(rt *libpod.Runtime, specg *specgen.SpecGenerator, contaierID s
 	specg.OverlayVolumes = overlay
 	specg.Mounts = conf.Spec.Mounts
 	specg.HostDeviceList = conf.DeviceHostSrc
+	mapSecurityConfig(conf, specg)
 	return c, nil
 }
+
+// mapSecurityConfig takes a libpod.ContainerSecurityConfig and converts it to a specgen.ContinerSecurityConfig
+func mapSecurityConfig(c *libpod.ContainerConfig, s *specgen.SpecGenerator) {
+	s.Privileged = c.Privileged
+	if len(c.LabelOpts) > 0 {
```

Comment:
Is this check necessary,  If you append an empty list, does it fail? Blow up?

Reply:
i will check with and without. I remember it adding a blank label and blowing up.

Reply:
it works without the check!